tasks/13.txt

```diff
@@ -1,3 +1,20 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+
+//task13
+db.products.aggregate(
+    [
+    { $match: {Discontinued:0}},
+    {
+        
+           $group: {
+             _id : "$ProductID", 
+              TotalOfCurrentProducts: { $sum: 1 } 
+        }
+    },
+    { $sort:{ProductID:-1}}
+    
+
+    ]
+ )
```

Comment:
we need 2 digit at the end here: TotalOfCurrentProducts  TotalOfDiscontinuedProducts.